### DISCUSSION TITLE
[2021.09.02] 괄호

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#92] [Algorithm]
[괄호](https://www.acmicpc.net/problem/9012)

**Comment by hyk339** [comment]
[괄호 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/baekjoon/BOJ_9012.java)

**Comment by chaeyeon4u** [comment]
[괄호 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_9012.java)

**Comment by pyunji** [comment]
[괄호 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_9012.java)

**Comment by leechang4** [comment]
[괄호 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/BOJ/BOJ_9012.java)
